appveyor.yml

```diff
@@ -30,6 +29,9 @@ install:
 
     # Install specified version of numpy and dependencies
     - "conda install --yes -c conda-forge numpy scipy nose setuptools ipython Cython sympy fastcache h5py matplotlib flake8 mock"
+    # HACK, FIXME BEFORE MERGING
```

Comment:
@ngoldbaum didn't you want to merge this?

Reply:
That’s a note to fix that issue before merging with master. We need to do a proper cykdtree release.